leapp/actors/__init__.py

```diff
@@ -303,7 +303,7 @@ def run(self, *args):
             pass
         except StopActorExecutionError as err:
             self.report_error(err.message, err.severity, err.details)
-        except StandardError as err:
+        except LeappRuntimeError as err:
             self.report_error(err.message, ErrorSeverity.FATAL)
```

Comment:
I wonder if we would want the stack trace as a detail, but you don't need to care about this here